docs/routing_conversation.md

```
@@ -0,0 +1,1215 @@
+
+    for i, r in enumerate(results):
+        if isinstance(r, Exception):
+            print(f"❌ Call {i} failed: {r}")
+        else:
+            print(f"✅ Call {i} answer: {r['choices'][0]['message']['content'][:60]}…")
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+**What this does**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)